report/finished/chapters/evaluation.tex

```diff
@@ -0,0 +1,14 @@
+\subsection{Engineering challenge}
+Performance of the robot during navigation on the setup shown in Fig. \ref{fig:factory_setup} was evaluated with the following criterias
+
+\begin{itemize}
+    \item Autonomy of the robot
+    \item Speed and navigation
+    \item Pickup / drop-off performance
+\end{itemize}
+
+The test were performedd at Luleå University of Technology at EISLAB with a local Arrowhead cloud and private network. The robot was able to navigate the factory floor and perform the action but moved slowly. This is because the Dynamixel motors used to translate the robot are slow. Communicating with Arrowhead was working and the test was proved successfull. A video with the whole run can be seen here: https://www.youtube.com/watch?v=YoAFspEC2no. 
+We can't do the evaluation without mentioning som limitations. The speed of the robot is mentioned above but also because of the line following algorithm the robot can get lost if it ever loses the line. This could be prevented with an external navigation system such as GPS, Ultra Wide Band (UWB) or simliar. 
+
```

Comment:
L: 10-11
spelling errors
performed
successful
some
similar

toughts: do we need to mention the reason for slow acuation is because the motors are design not to operate in wheelmode?